fireworks/core/firework.py

```diff
@@ -204,10 +208,9 @@ class Firework(FWSerializable):
     """
     A Firework is a workflow step and might be contain several Firetasks.
     """
-
     STATE_RANKS = {'ARCHIVED': -2, 'FIZZLED': -1, 'DEFUSED': 0, 'PAUSED': 0,
                    'WAITING': 1, 'READY': 2, 'RESERVED': 3, 'RUNNING': 4,
-                   'COMPLETED': 5}
+                   'COMPLETED': 5, 'CHECKPOINTED': 6}
```

Comment:
@computron comment: rank is used to set priority, so a checkpointed state should be less than completed